ext/com_dotnet/com_typeinfo.c

```diff
@@ -224,16 +223,7 @@ PHP_COM_DOTNET_API zend_result php_com_import_typelib(ITypeLib *TL, int mode, in
 
 				/* register the constant */
 				if (Z_TYPE(value) == IS_LONG) {
-					ZEND_CONSTANT_SET_FLAGS(&c, mode, 0);
-					ZVAL_LONG(&c.value, Z_LVAL(value));
-					if (mode & CONST_PERSISTENT) {
-						/* duplicate string in a persistent manner */
-						c.name = zend_string_dup(const_name, /* persistent */ true);
-						zend_string_release_ex(const_name, /* persistent */ false);
-					} else {
-						c.name = const_name;
-					}
-					zend_register_constant(&c);
+                    REGISTER_LONG_CONSTANT(const_name, Z_LVAL(value), mode);
```

Comment:
`const_name` is a `zend_string*`, but the first argument is supposed to be a `char*`.

Reply:
I reverted the change from this PR. It seems macros can't be use here as it's been reflected in the failed tests. I'm not sure yet how can I cast `zend_string*` to `char*`